flocx_market/api/bid.py

```diff
@@ -1,47 +1,36 @@
 from flask_restful import Resource
 from flask import request
-from flocx_market.db.sqlalchemy import api as dbapi
+import flocx_market.objects.bid as bid
 
 
 class Bid(Resource):
 
     @classmethod
     def get(cls, marketplace_bid_id=None):
-        if marketplace_bid_id is None:
```

Comment:
This change breaks the API. We expect `GET /bid` to return a list of all bids. With this change, it will instead return a 404 error.